### PR TITLE
Add exclude_class param to some 3D preprocessing layers. credit: lengzq

### PR DESCRIPTION
**Pull request by ianstenbit**
This change has already been made downstream and I'm syncing it here (since we don't have piper -> copybara set up).

### CODE REVIEW

File: keras_cv/layers/preprocessing_3d/frustum_random_dropping_points.py [hunk 1]
```diff
@@ -50,10 +50,18 @@ class FrustumRandomDroppingPoints(
...
     """
 
     def __init__(
-        self, r_distance, theta_width, phi_width, drop_rate=None, **kwargs
+        self,
+        r_distance,
+        theta_width,
+        phi_width,
+        drop_rate=None,
+        exclude_class=-1,
```
Comment (ianstenbit) [inline keras_cv/layers/preprocessing_3d/frustum_random_dropping_points.py:63]: This should probably use `None` instead of -1.

Comment (ianstenbit) [reply keras_cv/layers/preprocessing_3d/frustum_random_dropping_points.py]: Updated -- this now defaults to None.

File: keras_cv/layers/preprocessing_3d/frustum_random_dropping_points.py [hunk 2]
```diff
@@ -50,10 +50,18 @@ class FrustumRandomDroppingPoints(
       theta_width: A float scalar sets the theta width of a frustum.
       phi_width: A float scalar sets the phi width of a frustum.
       drop_rate: A float scalar sets the probability threshold for dropping the points.
+      exclude_class: A int scalar. When the last feature of a point == exclude_class, the point is not dropped.
```
[the remark this answers is not in the document]
Comment (ianstenbit) [reply keras_cv/layers/preprocessing_3d/frustum_random_dropping_points.py]: Good call -- updated to support a list (or `None`) throughout.

File: keras_cv/layers/preprocessing_3d/frustum_random_dropping_points.py [hunk 3]
```diff
@@ -60,11 +60,14 @@ def __init__(
         theta_width,
         phi_width,
         drop_rate=None,
-        exclude_class=-1,
```
Comment (LukeWood) [inline keras_cv/layers/preprocessing_3d/frustum_random_dropping_points.py]: nit: classes

**Review by LukeWood — APPROVED** [review]
Approvod with a nit, happy to delegate on this

**Comment by soma2000-lang** [comment]
@ianstenbit @LukeWood just want to educate myself on this part ,why we are doing this?

**Comment by ianstenbit** [comment]
> @ianstenbit @LukeWood just want to educate myself on this part ,why we are excluding the class params to some 3d processing layers?

The purpose here is to allow specifying classes that shouldn't be affected by these layers. There will be an upcoming blog post in the next few months about the usage of these layers which should help demonstrate the use case!

**Pull request merged**